R/population-sensitivity-analysis-settings.R

```diff
@@ -0,0 +1,103 @@
+#' @title PopulationSensitivityAnalysisSettings
+#' @description  R6 class for Population Sensitivity Analysis Settings
+#' @field variationRange variation range for sensitivity analysis
+#' @field numberOfCores number of cores for parallel computation
+#' @field quantileVec vector of quantiles to be calculated
+#' @field variableParameterPaths vector of paths of parameters to vary when performing sensitivity analysis
+#' @field pkParameterSelection list of selected PK parameters for sensitivity analysis
+PopulationSensitivityAnalysisSettings <- R6::R6Class(
```

Comment:
This feels kinds of weird to have a public member and then a function that updates the value

I'd suggest to change the implementation to use active keyword
Example (albeit complicated because we are wrapping .NET object here)
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/pk-parameter.R#L49

It's also described in details in the R6 class chapter from Wickham